bazelisk.go

```diff
@@ -566,6 +579,41 @@ func downloadBazel(fork string, version string, isCommit bool, directory string)
 	return destinationPath, nil
 }
 
+func copyFile(src, dst string, perm os.FileMode) (err error) {
+	data, err := ioutil.ReadFile(src)
+	if err != nil {
+		return err
+	}
+	return ioutil.WriteFile(dst, data, perm)
+}
+
+func linkLocalBazel(directory string, bazelPath string) (string, error) {
+	directoryName := dirForURL(bazelPath)
+	destinationDir := filepath.Join(directory, directoryName, "bin")
```

Comment:
please choose more a more descriptive name for at least one of directory / directoryName

Reply:
This is the only remaining comment from my side. Other than that: LGTM, great work!